### PR TITLE
Add projection management samples

### PR DESCRIPTION
**Pull request by George-Payne**
Related docs PR: https://github.com/EventStore/documentation/pull/424

### CODE REVIEW

**Review by ylorph — APPROVED** [review]

File: samples/projection-management.ts
Comment (oskardudycz) [inline samples/projection-management.ts:67]: **FIX:** Remove redundant spaces after `region` (here and in other places). They're rendered in docs and create too much space above the snippets. See e.g.
obraz

**Review by oskardudycz — APPROVED** [review]
@George-Payne, approving in advance, but please fix redundant new lines before merging (https://github.com/EventStore/EventStore-Client-NodeJS/pull/206/files#r693739520)

**Pull request merged**
